logging/src/test/java/com/nightlynexus/retrofit/logging/LoggingCallAdapterFactoryTest.java

```diff
@@ -119,55 +118,55 @@
     assertThat(latch.await(10, SECONDS)).isTrue();
   }
 
-  @Test public void enqueueLogsOnFailure() throws InterruptedException {
```

Comment:
this diff UI is a mess.

I removed the `@Ignore` from the 2.4.0 tests and deleted the 2.3.0 tests.
I added `enqueueDoesNotLogOnFailureExceptionsFatal` and `executeDoesNotLogOnFailureExceptionsFatal`.